### PR TITLE
update scatterelements and scatter_nd_impl

### PR DESCRIPTION
**Pull request by MPolaris**
update scatterelements and scatter_nd_impl codes from onnx repo.
scatterelements: https://github.com/onnx/onnx/blob/main/onnx/backend/test/case/node/scatterelements.py#L2
scatter_nd_impl: https://github.com/onnx/onnx/blob/main/onnx/backend/test/case/node/scatternd.py#L25

### CODE REVIEW

**Comment by ThanatosShinji** [comment]
Hi, thanks for your contribution! There are two questions about your commit. Please answer them, then I can start to merge.

**Comment by MPolaris** [comment]
Ok, What's your questions?

**Comment by ThanatosShinji** [comment]
You can see there are two unsolved conversations in your PR. And my questions are within them. Looking forward to your answers.

**Review by ThanatosShinji — CHANGES_REQUESTED** [review]

File: onnx_tool/node_profilers.py [hunk 1]
```diff
@@ -324,10 +324,14 @@ def scatter_nd_impl(data, indices, updates, reduction='none'):  # type: ignore
     output = numpy.copy(data)
     for i in numpy.ndindex(indices.shape[:-1]):
         # NOTE: The order of iteration in this loop is not specified.
-        if reduction == 'add':
+        if reduction == "add":
```
[the remark this answers is not in the document]
Comment (ThanatosShinji) [reply onnx_tool/node_profilers.py:327]: Format change from ONNX repo.

**Review by ThanatosShinji — APPROVED** [review]
I've confirmed these changes from ONNX repo.

**Comment by ThanatosShinji** [comment]
Thanks for your contribution!

**Pull request merged**